package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chili-publish/publisher-connector",
-  "version": "0.1.1",
+  "version": "0.1.0",
```

Comment:
Was it your intention to downgrade the version here?

Reply:
Seems like you're viewing an old commit, if you check out all commits you'll see that the opposite has been applied and the version is bumped to 0.1.2

Reply:
https://github.com/chili-publish/publisher-connector/pull/4/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

Reply:
Yep, I was watching a single commit :) Thx!